packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -34,11 +34,15 @@ export function newProject(options: OptionsSchema): Rule {
 
 		// TODO:
 		const defaultProjName = "IG Project";
-		let allOptionsProvided: boolean = false;
+		let nameProvided: boolean = false;
 
 		return chain([
 			(tree: Tree, _context: IgxSchematicContext): Observable<Tree> => {
 				return defer(async () => {
+					if (options.name) {
+						nameProvided = true;
+					}
+
 					if (!options.name || !prompt.nameIsValid(options.name)) {
```

Comment:
Not sure how this should behave if the provided name is not a valid one. With the current logic, if we pass an invalid name (e.g. `1111`), `nameProvided` will be set to `true` and the schematic will skip the `chooseActionLoop` part.

In the main cli, if an invalid name is provided, the process exits w/ an error message (defined in [new.ts](https://github.com/IgniteUI/igniteui-cli/blob/vNext/packages/cli/lib/commands/new.ts#L74))

Reply:
I think it is best to let the customer know the provided name is not valid and prompt him to provide a valid name. After a valid name is provided, we will show the Actions loop prompt.

Reply:
We have discussed it with @ViktorSlavov  and agreed it should exit the process as it may be invoked by a script and would never execute the prompt.